azurelinuxagent/common/protocol/goal_state.py

```diff
@@ -42,8 +43,10 @@ class GoalState(object):
     # fetch the goal state.
     #
     ContainerID = "00000000-0000-0000-0000-000000000000"
+    _IncarnationForCerts = None
+    _Certs = None
 
-    def __init__(self, wire_client, full_goal_state=False, base_incarnation=None):
+    def __init__(self, wire_client, ext_config_retriever, current_goal_state=None):
```

Comment:
current_goal_state doesn't seem to be used

Reply:
Removed